File: PASCAL/Cap03/c03ex08.pas
program c03ex08;

var
  A, B, X: Integer;

{
 Programa exemplo para operações de duas entradas
 com cálculo de adição e apresentação de resultado.
}

begin
  WriteLn('ADICAO DE NUMEROS');
  WriteLn;

  { Entrada de dados }

  Write('Entre o 1o. valor numerico: ');
  ReadLn(A);

  Write('Entre o 2o. valor numerico: ');
  ReadLn(B);

  { Processamento de dados }

  X := A + B; // Cálculo da adição
  
  { Saída de dados }

  WriteLn('Resultado = ', X);
  
  WriteLn;
  Write('Tecle <Enter> para encerrar... ');
  ReadLn;
end.

File: PHP/Cap03/c03ex08.php
<?php
// c02ex08.php

/*
 Programa exemplo para operações de duas entradas
 com cálculo de adição e apresentação de resultado.
*/

printf("ADICAO DE NUMEROS\n\n");

// Entrada de dados

printf("Entre o 1o. valor numerico inteiro: ");
$A = intval(trim(fgets(STDIN)));

printf("Entre o 2o. valor numerico inteiro: ");
$B = intval(trim(fgets(STDIN)));

// Processamento de dados

$X = $A + $B; // Cálculo da adição

// Saída de dados

printf("Resultado da adicao = %d\n", $X);

// Encerramento do programa

printf("\n");
printf("Tecle <Enter> para encerrar... ");
trim(fgets(STDIN));

?>
